geode-core/src/upgradeTest/java/org/apache/geode/security/AuthExpirationDUnitTest.java

```diff
@@ -320,10 +333,20 @@ public void cqOlderClientWillNotReAuthenticate() throws Exception {
     clientVM.invoke(() -> {
       // even user gets refreshed, the old client wouldn't be able to send in the new credentials
       UpdatableUserAuthInitialize.setUser("user2");
-      await().during(6, TimeUnit.SECONDS)
+      await().during(10, TimeUnit.SECONDS)
```

Comment:
Quick question: why does this call need to change from 6 seconds to 10 seconds?

Reply:
No particular reason, I guess I just want to wait a bit longer and 10 sounds better than 6.